### ISSUE TITLE
encryptModel.sh does not encrypt NMProperties

### ISSUE THREAD

**Issue by textanalyticsman** [oracle/weblogic-deploy-tooling#859]
After running encryptModel.sh I can see this.

`CustomIdentityKeyStorePassPhraseEncrypted: '{AES}RzBpZm1xTnhyb2Vjc0F2L2d4KzlRdWk4MXEwczEvb2ZzdTE4VVhKQTowRUQya1ZFc0t2Smw1d1BXOnZXczJXTWNIc1k0PQ=='`

However, when it comes to NMProperties, I can see

```
NMProperties:
        CrashRecoveryEnabled: true
        CustomIdentityKeyStorePassPhrase: cleartextpass
        CustomIdentityAlias: testkey
        CustomIdentityPrivateKeyPassPhrase: cleartextpass
        CustomIdentityKeyStoreFileName: /some/path/idstore.jks
        CustomIdentityKeyStoreType: JKS
        ListenPort: 6660
        KeyStores: CustomIdentityAndCustomTrust
        ListenAddress: 'machine01.pegasus.net'
        weblogic.StartScriptName: startWebLogic.sh
```
Is there a way to encrypt CustomIdentityKeyStorePassPhrase and CustomIdentityPrivateKeyPassPhrase?

Best regards,
TAM

**Comment by textanalyticsman** [comment]
Any ideas about this?

**Comment by CarolynRountree** [comment]
This is a bug and will be fixed in the next release.

**Comment by ddsharpe** [comment]
Fixed by #861
